ext/RastersArchGDALExt/RastersArchGDALExt.jl

```diff
@@ -0,0 +1,32 @@
+module RastersArchGDALExt
+
+import ArchGDAL,
+    DiskArrays,
+    Extents,
+    Missings
+
+using DimensionalData,
```

Comment:
Shouldn't these using statements only load the module?
using DimensionalData: DimensionalData as DD

Then you would not need the const lines below.

Reply:
I'm not sure that's actually shorter